boardswarm/src/gpio.rs

```diff
@@ -56,11 +56,9 @@ pub async fn start_provider(name: String, parameters: serde_yaml::Value, server:
                         let name = name.to_string_lossy().into_owned();
                         let mut properties = d.properties(name);
 
-                        info!(
-                            "GPIO?: {} - {} - {} - {:?}",
+                        debug!(
+                            "potential gpio device detected: {} - {:?}",
                             d.syspath().display(),
-                            registration.is_some(),
-                            properties.matches(&parameters.match_),
```

Comment:
You made me remember how this works again; the registration is not needed anymore indeed (this block wouldn't execute if there is a registration anyway) however the `properties.matches` is useful; Though it probably makes sense to debug  in the lower if/else block so it doesn't get evaluated twice; Essentially it would be good to get a debug message indicate *this* provider is ignoring a certain gpio device as it doesn't match 

Fwiw you'd also probably want to throw in an `#[instrument]` at the top to at least create a span with the name of the provider so it's clear *which* gpio provider is ignoring a given device  as i don't think that's currently included

Reply:
fwiw I am looking at how the gpio stuff works currently; this is where this PR came from and the reason why I put "potential" everywhere else and just confused myself :-)

So let me fix that as per what you have said here.